vcpkg/patches/oss/0002-vcpkg-dependencies-flags.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/scripts/toolchains/linux.cmake b/scripts/toolchains/linux.cmake
+index 70f224da..68d90470 100644
+--- a/scripts/toolchains/linux.cmake
++++ b/scripts/toolchains/linux.cmake
+@@ -17,6 +17,9 @@ if(NOT _CMAKE_IN_TRY_COMPILE)
```

Comment:
What is the meaning of _CMAKE_IN_TRY_COMPILE ?

Reply:
It's hard to say. I have to modify build flags globally for all packages but vcpkg doesn't have a way to do it (https://github.com/Microsoft/vcpkg/blob/master/docs/about/faq.md#can-i-use-my-ownspecific-flags-for-rebuilding-libs). So I found a workaround: https://github.com/Microsoft/vcpkg/issues/3577.